changelogs/fragments/2250-allow-keycloak-modules-to-take-token-as-param.yml

```
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - keycloak_* modules - allow the keycloak modules to take a token for the
+    authentification, the modules can take either a token or the credentials
```

Apparently "authentification" is a common misspelling of authentication, could you please change this here and other places in this PR?
```suggestion
    authentication, the modules can take either a token or the credentials
```